docs/ops/pooling/MaxUnpool.md

```diff
@@ -0,0 +1,60 @@
+## MaxUnpool <a name="MaxUnpool"></a> {#openvino_docs_ops_unpooling_MaxUnpool}
+
+**Versioned name**: *MaxUnpool*
+
+**Category**: *Unpooling*
+
+**Short description**: Performs max unpooling operation on input.
+
+**Attributes**: *Unooling* has no attributes.
+
+**Inputs**:
+  *   **1**: 4D pooling input tensor of shape `[N, C, H, W]` and type *T*. **Required.**
+  *   **2**: 4D pooling output tensor of shape `[N, C, H, W]` and type *T*. **Required.**
+  *   **3**: 4D input tensor of shape `[N, C, H, W]` and type *T*. **Required.**
+  *   **4**: 4D tensor of the required output shape `[N, C, H_out, W_out]` and type *T*. **Required.**
```

Comment:
I see that this is 2D Unpooling you are introducing but I found [1d](https://pytorch.org/docs/stable/generated/torch.nn.MaxUnpool1d.html), [2d](https://pytorch.org/docs/stable/generated/torch.nn.MaxUnpool2d.html), [3d](https://pytorch.org/docs/stable/generated/torch.nn.MaxUnpool3d.html) Unpooing in the pytorch, ND unpooling from the [onnx](https://github.com/onnx/onnx/blob/master/docs/Changelog.md#MaxUnpool-11) side. It would be great to have it ND way